save-frontend/src/main/kotlin/org/cqfn/save/frontend/utils/RequestUtils.kt

```diff
@@ -81,7 +81,14 @@ suspend fun request(url: String,
         credentials = credentials,
     )
 )
-    .await()
+    .await().also {
+        if (it.status == 401.toShort()) {
+            // if 401 - change current URL to the main page (with login screen)
+            // fixme: this should happen if there is no logged in user on FE
+            // otherwise 401 means that the user has insufficient permissions
```

Comment:
but insufficient permissions is more appropriate for 403, no?

Reply:
Yeah, I believe you're right. I meant that 401 might mean something different and not always cause redirect, but it probably always should.